server/x/doc.go

```diff
@@ -0,0 +1,2 @@
+// Packages x contains xpackages where each package's purpose to extend features of its origin.
+package x
```

Comment:
An interesting pattern!

Reply:
Thank you! I hope us to bring a @mattermost/x repo!